api/src/opentrons/protocol_reader/file_reader_writer.py

```diff
@@ -63,7 +68,24 @@ async def _read_file(
 
             if filename.lower().endswith(".json"):
                 try:
-                    data = parse_raw_as(BufferedJsonFileData, contents)  # type: ignore[arg-type]
+                    json_dict = json.loads(contents)
+                    if (
+                        json_dict.get("ordering") is not None
+                        and json_dict.get("wells") is not None
+                    ):
```

Comment:
Don't know why I didn't think of this during our review, but should we just use `key in Dict`?

```suggestion
                    if "ordering" in json_dict and "wells" in json_dict:
```